bin/region_mover.rb

```diff
@@ -493,11 +493,30 @@ def getFilename(options, targetServer, port)
   return filename
 end
 
+def getServersRSGroup(servers, rsgroup_admin, hostname, port)
+  rsgroup = rsgroup_admin.getRSGroupOfServer(Address.fromParts(hostname,
+                                             java.lang.Integer.parseInt(port)))
+  return rsgroup unless rsgroup.nil?
+  begin
+    server_name = getServerName(servers, hostname, port)
+    regions = getRegions(getConfiguration(), server_name)
+    if !regions.nil? && regions.empty?
+      exit 0
+    else
+      $LOG.warn('The server ' + hostname + ' belongs to no rsgroup. But still holds regions.')
+      exit 5
+    end
+  rescue java.io.IOException => e
+    $LOG.warn(e)
+    exit 0
```

Comment:
Since we want to exit gracefully, we should log it at INFO level

Reply:
Fixed in the latest commit.